src/extensions/flows/flow/flow.ts

```diff
@@ -69,7 +69,7 @@ export class Flow {
     const endTime = new Date();
     subject[isError ? 'error' : 'next']({
       type: 'flow:result',
-      id: capsule.component.id,
+      id: capsule.component.id.toString(),
```

Comment:
@qballer , I changed it before to be the BitId, not a string by purpose. 
We MUST NOT use strings. Here, in particular I need it to be BitId when I use it later.

Reply:
I see.